evap/static/css/evap.css

```diff
@@ -511,10 +511,19 @@ li.menu-indent {
 }
 
 .text-answer li,a {
-    word-break: break-word;
+    word-wrap: break-word;
```

Comment:
well, i commented "this seems like a merge fail". you didn't respond and when you said "done" i assumed you missed my comment. if it's a intended fix it's fine of course